src/models/EthAddress.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose')
+
+const EthAddressSchema = new mongoose.Schema({
```

Comment:
Seems a little weird to have the only other major field for EthAddress be emails. Maybe EthAddressEmails would be better, or maybe even AddressEmails. Not that important we can look at this again later.